typespec-extension/src/code-model-builder.ts

```diff
@@ -1529,114 +1488,157 @@ export class CodeModelBuilder {
     }
   }
 
+  private addParameterOrBodyToCodeModelRequest(
```

Comment:
`addParameterOrBodyPropertyTo...`?

I assume it be body property, not body.

Reply:
Yes, it's body property, will update.